client/src/components/DataFiles/DataFilesModals/DataFilesMakePublicModal.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import { SectionMessage } from '_common';
+
+const DataFilesMakePublicModal = () => {
+  const isOpen = useSelector(state => state.files.modals.makePublic);
+
+  const selected = useSelector(
+    state => state.files.modalProps.makePublic.selectedFile || {}
+  );
+
+  const dispatch = useDispatch();
+  const toggle = () => {
+    dispatch({
+      type: 'DATA_FILES_TOGGLE_MODAL',
+      payload: { operation: 'makePublic', props: {} }
+    });
+  };
+
+  const onClosed = () => {
+    dispatch({
+      type: 'DATA_FILES_SET_OPERATION_STATUS',
+      payload: { status: null, operation: 'makePublic' }
+    });
+  };
+
+  const makePublic = () => {
+    dispatch({
```

Comment:
@jarosenb Should we add another dispatch either here or in the saga to generate a notification (via `ADD_TOAST`) that says something like `Copy ${filename} to Public Data started`?

Reply:
I think closing the modal is enough verification. This is a very very rarely used feature on a single portal so I think our time would be better spent improving the UX elsewhere.

Reply:
Made a ticket: https://jira.tacc.utexas.edu/browse/FP-904